Blish HUD/Controls/Container.cs

```diff
@@ -126,6 +127,16 @@ public abstract class Container:Control, IEnumerable<Control> {
             return allDescendants;
         }
 
+        public void ClearChildren() {
```

Comment:
why is this approach better than

```cs
_children.ForEach(c => c.Parent = null);
_children.Clear();
```
?

Reply:
Changing the parent would modify the collection while it was enumerated (and throw an exception).  Setting `.Parent` the control calls `AddChild()` or `RemoveChild()` on the container.